object-construction-checker/tests/socket/ACSocketTest.java

```diff
@@ -214,19 +214,6 @@ void replaceVarWithNull(String address, int port) {
 
     }
 
-//    @EnsuresCalledMethodsIf(expression = "#1", methods = {"close"}, result = true)
```

Comment:
why is this deleted?

Reply:
I thought it's repetitive but it's not, so I added it again.
https://github.com/kelloggm/object-construction-checker/pull/291/commits/32c6c64d3fe2e77413bc1fadc0878fd07aea1cfd